packages/amplify-cli/src/commands/push.ts

```diff
@@ -16,7 +16,9 @@ import { collectDirectivesByTypeNames, readProjectConfiguration } from 'graphql-
 import path from 'path';
 import fs from 'fs-extra';
 
+
 const spinner = ora('');
+const TROUBLESHOOTING_PROJECT_URL = "https://docs.amplify.aws/project/troubleshooting/";
```

Comment:
Why is this URL being duplicated in this file?

Reply:
moved this to cli-core